feat: update openfga configuration (#140)
openfga/openfga.dev: docs/content/getting-started/setup-openfga.mdx

## Configuring The Server

> You may configure the OpenFGA sever with `config.yaml` file. The config.yaml is be specified in either 

Reviewer: can be specific

Author: Done

- `/etc/openfga`
- `$HOME/.openfga`
- `.` (i.e., the current working directory).

The OpenFGA will search for the configuration file in the above order.

> An example of configuration to use postgres as database and configured with preshared authentication keys "key1, key2" is:

Reviewer: More succintly,

"Here is a sample configuration that leverages postgres and preshared authentication:"

Author: Done

```yaml
database:

>   engine: postgres

Author: Updated

## Configuring The Server

> You may configure the OpenFGA sever with `config.yaml` file. The config.yaml can be specified in either 

Reviewer 2: Do you want backticks around the second `config.yaml`?
```suggestion
You may configure the OpenFGA sever with `config.yaml` file. The `config.yaml` can be specified in either 
```

- `/etc/openfga`
- `$HOME/.openfga`
- `.` (i.e., the current working directory).

The OpenFGA will search for the configuration file in the above order.

Here is a sample configuration that leverages postgres and preshared authentication:

```yaml
database:
  engine: postgres
  uri: postgres://user:password@localhost:5432/mydatabase
authn:
  method: preshared

>   preshared: [key1,key2]

Reviewer: This isn't quite right. Should be

```
authn:
    method: preshared
    preshared:
        keys: ["key1", "key2"]
```

Author: updated

## Configuring The Server

> You may configure the OpenFGA sever with `config.yaml` file. The `config.yaml` can be specified in either 

Reviewer 3: ```suggestion
You may configure the OpenFGA server with a `config.yaml` file. The `config.yaml` can be specified in either:
```

- `/etc/openfga`
- `$HOME/.openfga`
- `.` (i.e., the current working directory).

The OpenFGA will server search for the configuration file in the above order.

Here is a sample configuration to run OpenFGA with postgres and using a preshared key for authentication:

```yaml

> database:

Reviewer: Should be `datastore` now. We just refactored this to be consistent within OpenFGA with our vernacular.

```
datastore:
    engine: postgres
    uri: postgres://user:password@localhost:5432/mydatabase
```

- Update the configuration as follows:
  - Configure authentication on the server as explained above.
  - Enable TLS on the server.

>   - Enable production logging by changing the log format configuration to `json`.

Reviewer 4: ```suggestion
  - Enable production logging by changing the `log.format` in the [config.yaml](#configuring-the-server) configuration file to `json`.
```
